src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -40,6 +40,63 @@
                 }
             ]
         },
+        {
+            "path": "/v1/debug/self_test/start",
+            "operations": [
+                {
+                    "method": "POST",
+                    "summary": "Start self test",
+                    "nickname": "self_test_start",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": [],
+                    "responses": {
+                        "200": {
+                            "description": "OK",
+                            "schema": "json"
+                        },
+                        "500": {
+                            "description": "Test failed to start",
```

Comment:
500 means something totally unexpected: if the test cannot start right now for known reasons, that would be a 503, whereas if we get a 500 we cannot say anything about why (it's not something we anticipated)